domain-model-assistant/Assets/Components/Scripts/TextBox.cs

```diff
@@ -13,16 +15,38 @@ public string ID
     { get; set; }
     public bool isHighlightedtext
     { get; set; }
+    public string name; //second substring of attribute
 
+    public int typeId;
+    
+    // public bool isChecked;
+    
     float holdTimer2 = 0;
     bool hold2 = false;
+
+    void Awake()
+    {
+        _diagram = GetComponentInParent<Diagram>();
+    }
+
     void Start()
     {
 
     }
 
     void Update()
     {
+        if(Input.GetKeyDown(KeyCode.Return)){
+            Debug.Log("Enter");
+            string _id = this.ID;
+             if(isValid() && _id.Equals("-1")){
```

Comment:
Formatting here does not match the default C# Style used in the rest of the codebase

Reply:
Fixed